ccm_web/client/src/components/ValidationErrorTable.tsx

```diff
@@ -0,0 +1,128 @@
+import React, { useEffect, useState } from 'react'
+import { createStyles, Paper, Table, TableBody, TableCell, TableContainer, TableFooter, TableHead, TablePagination, TableRow, Theme, withStyles } from '@material-ui/core'
+import { grey } from '@material-ui/core/colors'
+import { TablePaginationActions } from './TablePagination'
+
+const StyledTableCell = withStyles((theme: Theme) =>
+  createStyles({
+    head: {
+      backgroundColor: grey[100],
+      color: theme.palette.common.black,
+      fontWeight: 'bold'
+    },
+    body: {
+      fontSize: 24
+    }
+  })
+)(TableCell)
+
+interface ValidationError {
+  rowNumber: number
+  message: string
+}
+
+interface ErrorTableProps {
+  invalidations: ValidationError[]
+}
+
+interface TableHeaderColumnInfoShouldUseMatUIType {
+  id: keyof ValidationError
+  label: string
+  minWidth: number
+  align?: 'left' | 'right' | undefined
+}
+
+const columns: TableHeaderColumnInfoShouldUseMatUIType[] = [
+  { id: 'rowNumber', label: 'Row Number', minWidth: 25 },
+  { id: 'message', label: 'Error', minWidth: 100 }
+]
+
+function ErrorTable (props: ErrorTableProps): JSX.Element {
+  const [tableRows, setTableRows] = useState<ValidationError[]>([])
+  const [page, setPage] = React.useState<number>(0)
+  const [rowsPerPage, setRowsPerPage] = React.useState(5)
```

Comment:
See comments on `ConfirmationTable` about maybe not needing the dynamic `rowsPerPage` functionality (and handler `handleChangeRowsPerPage` below).